### ISSUE TITLE
URGENT: First line of news body appears twice on search result

### ISSUE THREAD

**Issue by damianino1** [microsoft-search/pnp-modern-search#1985]
Hello,

Usually we're starting the body of our news pages with a date:
![image](https://user-images.githubusercontent.com/102507233/161067439-cc732caa-cb31-491a-b6c0-cefce5b97473.png)

Unfortunately, something is wrong with the result web part as the first line of news page body on the search result is often listed twice :

![image](https://user-images.githubusercontent.com/102507233/161066809-6424915f-7fe7-4425-8de1-d9f7b0d6b3ad.png)

or twice and with some ID:

![image](https://user-images.githubusercontent.com/102507233/161073177-2e686233-0a69-4c95-a536-e67061dcc963.png)

But as you see, this happens also not on each news page for ex. the middle on is not affected. We also using versioning in English and German. Both are configured exact on the same way except the additional line for filter for the language:

![image](https://user-images.githubusercontent.com/102507233/161063376-f5de1472-f765-4800-b670-013ebe04c945.png)

but only 95% of German news pages are affected. Strange fact is, that if the news page body does not start with a date, then this issue does not appear.

German news pages with no date in body will be shown correctly:

![image](https://user-images.githubusercontent.com/102507233/161065562-736b8fb5-9bb3-472e-91b6-bc9a1db204a1.png)

Or all our English news as well, even they begin with a date:

![image](https://user-images.githubusercontent.com/102507233/161066849-8f1d6f81-0796-429f-b050-386577b071dd.png)

I already tried to re-build the webparts from scratch but with the same result. I also re-indexed the page library and the whole site again but it didn't help. The standard out-of-the-box sharepoint "show all" news web part doesn't have this issue.

**It is very critical for my as we're using it on productive Intranet for about 10k users, so it us really urgent.**

Attached my config json file: [https://drive.google.com/file/d/184c3IGw__J_iX-41YF_hVnXdzUbznSZO/view?usp=sharing](url)

Any suggestions?

Thanks in advance.

**Comment by FranckyC** [comment]
Hi @damianino1, first of all we need to identify if it is a WebPart issue or an search index issue. To do that, simply inspect the call `postquery` in the network tab and especially the response and the property `HitHighlightedSummary`. I see in you configuration you are using the `Summary` slot with default mapping so the value should come from this property.

If the content is the same as you see in the card, then this is not a WebPart issue and you should open a ticket with Microsoft.

Also you did not mention the version you are using. There was an similar issue raised few days ago. See https://github.com/microsoft-search/pnp-modern-search/discussions/1869.

In the meantime, as a workaround, what I can suggest it to use an other field like the 'Description' on the pages so you can control the output. 

![image](https://user-images.githubusercontent.com/5659122/161077913-58d0a4bf-f5d8-41e4-8f71-31baa95b1c32.png)

Of course it requires to edit all the pages and index again + change the `Summary` slot to use `Description` instead of `HitHighlightedSummary`

Hope it helps.

**Comment by damianino1** [comment]
@FranckyC : I am using version 4.2.3.

HitHighlightedSummary also includes this issue:
`{"Key":"HitHighlightedSummary","Value":" \u200b\u200b\u200b\u200b\u200b\u200b\u200b 1. April 20221. April 2022. Vergangene Woche`

I think that this is a Microsoft's issue as the home page news webpart as well as the out-of-the box show this values correctly:

Standard News webpart:
![image](https://user-images.githubusercontent.com/102507233/161234310-8cfe6c54-e4f3-4f9e-bf9b-1eeaa7e4548e.png)

Standard News Archive:
![image](https://user-images.githubusercontent.com/102507233/161233966-21183f2c-dc76-4f5b-b9e5-92447b535875.png)

Do you have any recommendation what can I do?

**Comment by FranckyC** [comment]
You mean the `HitHighlightedSummary` property value from the `postquery` call returns incorrect data? If it is the case, then it is an API issue as we don't know what kind of stripping process the OOTB Web Parts are doing. @wobba is MS know about this issue and fixed it client side by any chance?

**Comment by wobba** [comment]
Closing this one as there are many ways to get a summary. If you use the `Description` managed property as the description, and also for summary description, then you likely won't have this issue. And it is a back-end issue as it's the backend generating the hithighlighted summary, not the web part.

**Comment by damianino1** [comment]
@wobba we're fullfilling description field on each our news page. I also deactivated `HitHighlightedSummary` and activated `Description` on Selected properties in PnP - Search Results webpart - but still the same behaviour.

What I am doing wrong ?

**Comment by wobba** [comment]
The web parts aren't configured to ask for `Description` as the hithighlighted property - which is one way to go about this. You can however put `Description` as the property you want to display instead of the summary.

**Comment by damianino1** [comment]
@wobba @FranckyC I can't find `Description` at Layout Slots. Does it mean, that I have to set up a new managed property on Admin Center for it? If yes what is the exact name of this property?  

I am a little lost, please provide me an additional advice or suggestion for better understanding

**Comment by FranckyC** [comment]
You need to first add it to selected properties:

![image](https://user-images.githubusercontent.com/5659122/161751230-63e5b313-6902-46b5-869a-35d5654f1aa4.png)

Then use it as slot:

![image](https://user-images.githubusercontent.com/5659122/161751280-26dc5c0d-008c-42d7-a197-f4396117ef6a.png)

**Comment by damianino1** [comment]
@FranckyC thanks! One word says more the thousends words :)

Anyway I am stacking at one more problem. I changed it on the way described by you and now my search result looks as it should in the page edit mode, but as soon as I save and re-publish, it shows the old version again. I also tried with InPrivate and got the same result. Is it maybe a bug in the 4.2.3? Do you have maybe any suggestion for it, because it still stopps me on the way to resolve it.

**Comment by damianino1** [comment]
@FranckyC thanks, but hard refresh (which I tried at very first), re-indexing of library or whole site collection, rebuilding the whole webparts neighter updating the solution did NOT resolve this issue. Edit mode still showed the changes but after publishing it was always the same "buggy" 'frozen' version. 

I spent my half friday on it and finally could solve it by exporting search result setting of the english version (which works fine and wasn't affected by the 'double' date issue) and imported it into german version of webpart.

Thank you very much for supporting me on it and maybe this topic will help someone in the future.
